docs/source/models/gbd2017/causes/cause_model_template.rst

```diff
@@ -91,125 +82,193 @@ Disease Description
+     -
+     -
+   * - Female only
+     -
+     -
+   * - YLL only
+     -
+     -
+   * - YLD only
+     -
+     -
+   * - YLL age group start
+     -
+     -
+   * - YLL age group end
+     -
+     -
+   * - YLD age group start
+     -
+     -
+   * - YLD age group end
+     -
+     -
+
+
+Vivarium Modeling Strategy
+--------------------------
 
-      Point people to yld and cod appendices for their diseases and link to
-      http://ghdx.healthdata.org/gbd-2017/code which can serve as an annoying
-      to read but authoritative source on how the cause was actually modeled.
+Scope
++++++
 
+.. todo::
 
-Cause Hierarchy
-+++++++++++++++
+  Describe which aspects of the disease this cause model is designed to
+  simulate, and which aspects it is **not** designed to simulate.
+
+Assumptions and Limitations
++++++++++++++++++++++++++++
+
+.. todo::
+
+  Describe the clinical and mathematical assumptions made for this cause model,
+  and the limitations these assumptions impose on the applicability of the
+  model.
+
+.. admonition:: Todo for template development
+
+  What else goes in the `Vivarium Modeling Strategy` section? Should there be
+  any other subsections besides `Scope` and `Assumptions and Limitations`?
 
 Cause Model Diagram
 -------------------
 
-State and Transition Data Tables
---------------------------------
+Data Description
+----------------
 
 .. admonition:: Todo for template development
 
-    - Decide on section heading names and structure. E.g. should this section be
-      called "Data Description" instead of "State and Transition Data Tables,"
-      like it was originally?
+  Decide on section heading name(s) and structure for the data section. See
+  comments by @Beatrix and @Nathaniel in `Pull Request 64`_ discussing the
+  following questions:
 
-      In `Pull Request 91`_, @James said:
+  - Should this section be called "Data Description" as it is now, or would
+    something like "State and Transition Data Tables" be more descriptive?
 
-        I think `Data Description` is fine.
+    In `Pull Request 91`_, @James said:
+
+      I think `Data Description` is fine.
 
-    - Is there anything else that belongs in a "Data Description" section
-      besides the data tables?
+  - Should this section have subsections (such as "State and Transition Data
+    Tables")? Is there anything else that belongs in a "Data Description"
+    section besides the data tables?
 
-      In `Pull Request 91`_, @Yongquan said:
+    In `Pull Request 91`_, @Yongquan said:
 
-        one data-relevant section needed for more complicated models is
-        non-standard data sources where external calculation occurs.
+      one data-relevant section needed for more complicated models is
+      non-standard data sources where external calculation occurs.
 
-    - Add instructions in other sections, including:
+    Does "non-standard data sources" warrant a subsection, even if we don't need
+    it for all cause models?
```

Comment:
I think we can handle non-standard data sources on an ad hoc basis and probably don't need a subsection for it given my understanding that it is not necessary for most causes? If I'm wrong in this assumption and it is in fact necessary for most causes, then I think we can include a subsection.

Reply:
The non-standard data sources should be handled in table 4 (the data definitions).  The point of the table layouts we came up with was to be flexible enough to handle all known use cases.